src/configuration/services/mysql.md

```diff
@@ -277,3 +277,6 @@ Errors such as "PDO Exception 'MySQL server has gone away'" are usually simply t
 The current disk usage can be checked using the CLI command `platform db:size`. Because of issues with the way InnoDB reports its size, this can out by up to 20%. As table space can grow rapidly, *it is usually advisable to make your database mount size twice the size reported by the `db:size` command*.
 
 You are encouraged to add a [low-disk warning notification](/administration/integrations/notifications.html#low-disk-warning) to proactively warn of low disk space before it becomes an issue.
+
+Another issue might be in context of long running worker processes that try to re-use a timed out connection. In such cases it can happen to see the same error popping up in your logs. If possible try to establish a new connection in your application or - in case of workers - ensure a shorter execution/idle time or enforce some sort of restart by exiting the current command. 
+In PHP environments you can also try to increase the `mysql.connect_timeout` and/or `default_socket_timeout` in your [php.ini](/languages/php/ini.html).
```

Comment:
You are likely hitting the mysql 10 minutes timeout, which means if anything, you want to _lower_ the timeout on the client side, to make sure it's trying to ping the connection again sooner.

Reply:
As far as I can see the default php-ini's `default_socket_timeout` sits at 60 seconds. Shouldn't this do the trick then?

Reply:
I'm guessing this timeout is a TCP timeout for the initial connection. You likely want a "MySQL idle timeout", just maintaining the TCP connection is not enough.

Reply:
Yeah, I see. Thanks for the feedback. I was just wondering that this kind of incident is not documented anywhere. It would have helped me while troubleshooting.
Shall I remove the tip for the timeout-adjustments? I cannot really provide a tip for configuring this properly right now - except for wrapping the failing code and try to reconnect.

Reply:
I... honestly don't know. I'm bad at documentation...